### PR TITLE
Raise intents

### PR DESCRIPTION
**Pull request by venetagrigorova**

### CODE REVIEW

File: packages/ts-fdc3-oms/src/components/Portfolio/Portfolio.tsx [hunk 1]
```diff
@@ -161,29 +161,26 @@ const Portfolio = (): JSX.Element => {
...
+                .raiseIntent('NewOrder', {
+                    type: 'fdc3.order',
+                    order: {
+                        type: 'fdc3.order',
+                        side: side,
+                        instrument: instrument,
+                        quantity: quantity,
+                        notes: selectedFund?.name
+                    }
+                })
```
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/src/components/Portfolio/Portfolio.tsx]: you must add the correct "target" - if it is reause in glue.intents - it means that the target is the current Portfolio app instance, or another open app (like TradeHistory) for example, but we gotta put the correct one. Syntax is { appId: "/app id in json condfig definition/"}

Comment (bogdandinchev) [reply packages/ts-fdc3-oms/src/components/Portfolio/Portfolio.tsx]: in this case i would bet it targets the Orders app, but you need to double check

File: packages/ts-fdc3-oms/acme-service/src/acmeService.ts [hunk 2]
```diff
@@ -304,45 +306,30 @@ class AcmeService {
...
 
     private raiseViewOrderHistory = async (context: any): Promise<void> => {
         if (await this.shouldRaiseIntent()) {
-            this.glue.intents
-                .raise({
-                    intent: ViewOrderHistoryIntent,
-                    context,
-                    target: 'reuse',
-                })
+            this.fdc3?.raiseIntent(ViewOrderHistoryIntent, context)
```
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/acme-service/src/acmeService.ts]: I don't think the context is in the right format off the bat - search '                    this.raiseViewOrderHistory' in files and see. You would have to normalize the data:

this.raiseViewOrderHistory({
                        data: {
                            securityId: {
                                ticker: ticker,
                                bbgExchange: exchange,
                            },
                            clientId: '',
                        },
                    })
like remove the data prop in this case

File: packages/ts-fdc3-oms/acme-service/src/acmeService.ts [hunk 3]
```diff
@@ -304,45 +306,30 @@ class AcmeService {
...
-            (i: any) => i.name === ViewOrderHistoryIntent
-        )
-
-        const handler = intent?.handlers?.find(
-            (h: any) => h.type === 'instance'
-        )
-
-        return handler && handler.applicationName === 'acme-oms-order-history'
+        const intents = await this.fdc3.findIntent(ViewOrderHistoryIntent)
+        console.log('intents', intents)
```
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/acme-service/src/acmeService.ts]: remove logs

File: packages/ts-fdc3-oms/acme-service/dist/acme-service-0.1.0.js [hunk 4]
```diff
@@ -1,3 +1,1485 @@
...
+(function webpackUniversalModuleDefinition(root, factory) {
+	if(typeof exports === 'object' && typeof module === 'object')
```
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/acme-service/dist/acme-service-0.1.0.js:2]: Did you just format the files in dist? I don't think they should be changed manually - they would change themselves after a build is triggered

File: packages/ts-fdc3-oms/acme-service/src/acmeService.ts [hunk 5]
```diff
@@ -304,45 +306,30 @@ class AcmeService {
...
-        this.glue?.intents.raise({
-            intent: 'NewOrder',
-            context: {
+        this.fdc3?.raiseIntent('NewOrder', {
                 type: 'fdc3.order',
-                data: order,
-            },
-            target: 'reuse',
-        })
+                order,
```
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/acme-service/src/acmeService.ts]: check 'reuse'

File: packages/ts-fdc3-oms/acme-service/src/acmeService.ts (hunk 2 above)
Comment (bogdandinchev) [inline packages/ts-fdc3-oms/acme-service/src/acmeService.ts]: also check 'reuse'

**Review by bogdandinchev — CHANGES_REQUESTED** [review]

File: packages/ts-fdc3-oms/acme-service/dist/acme-service-0.1.0.js (hunk 4 above)
Comment (venetagrigorova) [reply packages/ts-fdc3-oms/acme-service/dist/acme-service-0.1.0.js:2]: no i didn't format them. i just reran the build and they were updated

**Pull request merged**